CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 1.3.TBD - TBD
```

Comment:
I assume that this would be a minor version bump to 1.4.0 since it is adding new functionality.